source/features/edit-comments-faster.tsx

```diff
@@ -7,13 +7,13 @@ import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
-function canEditDiscussion(): boolean {
-	return pageDetect.isDiscussion() && select.exists('[title^="You are a maintainer"], [title^="You are a collaborator"]');
+function canEditComments(): boolean {
+	return select.exists('[aria-label^="You are the owner"], [title^="You are a maintainer"], [title^="You are a collaborator"]');
```

Comment:
Based on https://github.com/sindresorhus/refined-github/pull/4415#issuecomment-850970454, you'll also need `[aria-label^="You have been invited to collaborate"]`, i.e.

```suggestion
	return select.exists('[aria-label^="You have been invited to collaborate"], [aria-label^="You are the owner"], [title^="You are a maintainer"], [title^="You are a collaborator"]');
```

for issues/PRs of repos you are a Collaborator (so you can edit all comments).

&nbsp;
Plus, there's still the uncovered case when you are the owner/maintainer/collaborator of a repo (so you can edit all comments), and there's an issue/PR/discussion where you haven't made a comment yet (so the checks don't apply), 
but you can edit the title (=even if you're not the author).

Reply:
Will add that check (most probably by checking if the user can edit the title or a settings tab is present), thanks